tests/app-test.js

```diff
@@ -1,10 +1,77 @@
-/* global describe, beforeEach, afterEach, it */
+/* global describe, beforeEach, afterEach */
 import { expect } from 'chai';
+import $ from 'jquery';
+import sinon from 'sinon';
+import it from './it-will';
```

Comment:
I was actually thinking maybe we want to move this to `it.will` so we could still use `it` if we absolutely need to